editor/src/clj/editor/engine_profiler.clj

```diff
@@ -51,14 +51,22 @@
                     "text/html")]
     mime-type))
 
-(defn- read-resource [req]
+(defn- try-local-path [req]
   (let [path (str "." (:url req)) ; Make it a relative path as it starts with a "/"
-        mime-type (get-mime-type path)
-        resource (io/resource path)
-        data (if (= resource nil)
-               nil
-               (slurp resource))
-        code (if (= data nil) 404 200)]
+        resource (io/resource path)]
+    resource))
+
+(defn- try-package-path [req]
+  (let [path (subs (:url req) 1) ; remove the initial "/"
```

Comment:
If you're loading directly from the package, the paths should be like: "path/to/file"